src/components/Chart.tsx

```diff
@@ -1,68 +1,120 @@
-import React from "react";
-import {
-  LineChart,
-  Line,
-  XAxis,
-  YAxis,
-  CartesianGrid,
-  Tooltip,
-  Legend,
-  ResponsiveContainer,
-} from "recharts";
+import SuccessFailChart from "./SuccessFailChart";
+import SuccessChannelsChart from "./SuccessChannelsChart";
+import FailChannelsChart from "./FailChannelChart";
+import { useState, useEffect } from "react";
 
-const data = [
-  {
-    name: "Page A",
-    uv: 4000,
-    pv: 2400,
-    amt: 2400,
-  },
-  {
-    name: "Page B",
-    uv: 3000,
-    pv: 1398,
-    amt: 2210,
-  },
-  {
-    name: "Page C",
-    uv: 2000,
-    pv: 9800,
-    amt: 2290,
-  },
-  {
-    name: "Page D",
-    uv: 2780,
-    pv: 3908,
-    amt: 2000,
-  },
-  {
-    name: "Page E",
-    uv: 1890,
-    pv: 4800,
-    amt: 2181,
-  },
-  {
-    name: "Page F",
-    uv: 2390,
-    pv: 3800,
-    amt: 2500,
-  },
-  {
-    name: "Page G",
-    uv: 3490,
-    pv: 4300,
-    amt: 2100,
-  },
-];
+import BackendSDK from "./../../../backend-sdk/src/index.ts";
+const apiUrl = import.meta.env.VITE_HTTP_GATEWAY;
+const naas = new BackendSDK("secretkey1", apiUrl!);
```

Comment:
I'd like to move away from importing the backend-sdk and instead fetch data from the API endpoints directly. Here is an example `useEffect` you can use to replace the functionality we're currently getting through the backend-sdk.

Also please create a `VITE_API_KEY` variable in the .env file.

```
  useEffect(() => {
    const getNotificationLogs = async () => {
      try {
        const apiUrl = import.meta.env.VITE_HTTP_GATEWAY;
        const apiSecret = import.meta.env.VITE_API_KEY;

        const url = apiUrl + `/notifications`;

        let response = await fetch(url, {
          method: "GET",
          headers: {
            Authorization: apiSecret,
          },
        });

        if (!response.ok) {
          throw new Error(`HTTP error: ${response.status}`);
        }

        const fetchedLogs = await response.json();
        setLogs(fetchedLogs);
      } catch (error) {
        console.log("Error fetching notification logs", error);
      }
    };

    getNotificationLogs();
  }, []);
```